ledgerwallet/manifest.py

```diff
@@ -100,73 +96,27 @@ def is_power2(n):
 
 
 class AppManifest(object):
```

Comment:
As this now looks like an interface, this could be properly defined as an abstract class (`abs.ABC`), with `@abstractmethod` and `@abstractproperty` decorators.